{"hierarchy":{"paths":[["doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber","doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception"]]},"sections":[],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/Error-Implementations"},"topicSections":[{"title":"Instance Properties","identifiers":["doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/localizedDescription"],"generated":true,"anchor":"Instance-Properties"}],"schemaVersion":{"major":0,"minor":3,"patch":0},"metadata":{"roleHeading":"API Collection","title":"Error Implementations","modules":[{"name":"SwiftKyber"}],"role":"collectionGroup"},"kind":"article","variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftkyber\/exception\/error-implementations"]}],"references":{"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception":{"type":"topic","abstract":[{"text":"The Kyber exceptions","type":"text"}],"kind":"symbol","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"Exception"}],"url":"\/documentation\/swiftkyber\/exception","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"Exception"}],"title":"Exception"},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber":{"abstract":[{"type":"text","text":"Module-Lattice-Based Key-Encapsulation Mechanism Standard"}],"kind":"symbol","type":"topic","role":"collection","title":"SwiftKyber","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber","url":"\/documentation\/swiftkyber"},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception/localizedDescription":{"kind":"symbol","url":"\/documentation\/swiftkyber\/exception\/localizeddescription","title":"localizedDescription","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"localizedDescription","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"abstract":[],"type":"topic","role":"symbol","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/localizedDescription"}}}